dandi/tests/test_validate.py

```diff
@@ -1,34 +1,61 @@
-from glob import glob
 import json
 import os
 import pathlib
 
 import pytest
 
 from .fixtures import BIDS_TESTDATA_SELECTION
-from ..validate import validate_bids
+from ..validate import validate
+
+
+def test_validate_nwb_error(simple3_nwb):
+    """
+    Do we fail on critical NWB validation errors?
+    """
+
+    validation_result = validate(simple3_nwb)
+    assert len([i for i in validation_result if i.severity]) > 0
 
 
 @pytest.mark.parametrize("dataset", BIDS_TESTDATA_SELECTION)
 def test_validate_bids(bids_examples, tmp_path, dataset):
 
     selected_dataset = os.path.join(bids_examples, dataset)
-    validation_result = validate_bids(selected_dataset, report=True)
+    validation_result = validate(selected_dataset)
     for i in validation_result:
         assert i.severity is None
 
 
-def test_report_path(bids_examples, tmp_path):
+def test_validate_bids_onefile(bids_error_examples, tmp_path):
+    """
+    Dedicated test using to single-file validation.
```

Comment:
note to myself: unlike `nose`, `pytest` doesn't use docstring as the "name" of the test while running testing so there is no obfuscation and we are indeed "open" to provide docstrings giving details on tests. good.